test/controllers/communicator.js

```diff
@@ -1,30 +1,35 @@
 const request = require('supertest');
```

Comment:
Were these changes necessary to fix the tests? If not, I know it's very tempting but I would recommend _not_ touching files when it's out of scope for the PR, to make the reviewers' job easier :-)

Reply:
Same for `languages.js`, `settings.js`, etc.

Reply:
really sorry for that! and it's better when I make the requested changes too... 👎 Was very tempting true and I did it to be sure that the nodemailer mock work properly but mock it is no a requirement of this PR. how do you recommend fix this? thanks for your time Simon